.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -360,6 +401,9 @@ stages:
             echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
             echo '##vso[task.setvariable variable=OS_SKU]Ubuntu'
             echo '##vso[task.setvariable variable=OS_VERSION]18.04'
+            echo '##vso[task.setvariable variable=IMG_PUBLISHER]Canonical'
```

Comment:
arguably you could set these as the defaults and only override for 20.04, but can follow up